.changeset/poor-pants-tan.md

```diff
@@ -0,0 +1,5 @@
+---
+"@keystone-next/fields": patch
+---
+
+Fix defaultValue types for autoIncrement and integer fields
```

Comment:
```suggestion
Fixed defaultValue types for `AutoIncrement` and `Integer` fields.
```

Past tense, full sentence. Leads to better changelogs :-)

Reply:
Thanks